packages/components/ovh-at-internet/src/utils.ts

```diff
@@ -1,6 +1,44 @@
 export function debug(...args: any[]) {
-  if (!window.test && //temporarily  fix jest bug
-    window.localStorage?.getItem('MANAGER_TRACKING_DEBUG')) {
+  if (
+    !window.test && // temporarily  fix jest bug
+    window.localStorage?.getItem('MANAGER_TRACKING_DEBUG')
+  ) {
     console.debug(...args);
   }
 }
+
+export const isEmpty = (obj: Record<string, unknown>) => {
+  if (obj?.length || obj?.size) return false;
+  if (typeof obj !== 'object') return true;
+  return !Object.values(obj).length;
+};
+
+export const capitalize = (str: string) => {
+  return `${str.charAt(0).toUpperCase()}${str.slice(1).toLowerCase()}`;
+};
+
+export const mapValues = (
+  obj: Record<string, unknown>,
+  fn: (val: unknown, key: string, obj: Record<string, unknown>) => unknown,
+) =>
+  Object.fromEntries(
+    Object.entries(obj).map(([key, val]) => [key, fn(val, key, obj)]),
+  );
+
+export const getRandomString = (len: number): string => {
+  const alphabet =
+    'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+  return new Array(len)
+    .map(() => alphabet.charAt(Math.floor(Math.random() * alphabet.length)))
+    .join('');
+};
+
+/**
+ * Returns an unique ID (used by orders and carts)
+ * To ensure that the ID is unique, we generate it from current milliseconds
+ * and concat a random string to avoid any collisions.
+ */
+export const getUniqueId = (): string => {
+  // unique key : current milliseconds in base 36 concatenated with 8 random chars
+  return new Date().valueOf().toString(36) + getRandomString(8);
```

Comment:
why not used uuid instead ?

Reply:
I did not changed this function, only deleted the file from `utils/index.ts` and moved it all into this `utils.ts` file